var/spack/repos/builtin/packages/dray/package.py

```diff
@@ -0,0 +1,332 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+import os
+import socket
+
+import llnl.util.tty as tty
+from os import environ as env
```

Comment:
Doesn't Spack already add this to the module? In `build_environment.py`:
```python
    # easy shortcut to os.environ                                               
    m.env = os.environ   
```

Reply:
Perhaps this is now implicitly there?

q: with the way flake8 is leveraged by spack -- would it catch an issue if we drop the import and  `env` wasn't satisfied?

Reply:
Flake8 will still pass since it actually follows all of the imports.

Reply:
great, thought that might be the case -- thanks for verifying.
I removed this import